src/main/java/core/basesyntax/strategy/impl/ApplySupplay.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitNegotiation;
+import core.basesyntax.strategy.UnaryOperation;
+
+public class ApplySupplay implements UnaryOperation {
+    private FruitNegotiation myFruit;
+
+    public ApplySupplay(FruitNegotiation fruit) {
+        this.myFruit = fruit;
+    }
```

Comment:
this transaction should be just passed as a method parameter instead
same for all of the `UnaryOperations`